Rubberduck.Core/UI/Command/MenuItems/CommandBars/SerializeProjectsCommandMenuItem.cs

```diff
@@ -77,7 +78,12 @@ protected override void OnExecute(object parameter)
                 _serializationProvider.SerializeProject(library);
             }
 
-#if TRACE_COM_SAFE
+            SerializeComSafe();
+        }
+
+        [Conditional("TRACE_COM_SAFE")]
+        private void SerializeComSafe()
+        {
             //This block must be inside a conditional compilation block because the Serialize method 
             //called is conditionally compiled and available only if the compilation constant TRACE_COM_SAFE is set.
```

Comment:
still relevant?

Reply:
Yes. Though it has moved out to the `Conditional("TRACE_COM_SAFE")` and the `Serialize` is no longer conditionally compiled, a build without the flag will be a no-op:

https://github.com/rubberduck-vba/Rubberduck/pull/5182/files#diff-3b3c40ef6fbf25159c4baf172e7d3ca8R33-R39

(GH doesn't want to inline the code. 😞 )